### PR TITLE
Feature/list of approved

### PR DESCRIPTION
**Pull request by joaoprferreira**
## Description

Construir os serviços para consumir a lista de alunos aprovados.

## Developer Checks

- [ ] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [ ] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -21,8 +21,29 @@ import {
...
+export const getStaticProps: GetStaticProps = async () => {
+  const approvedList = await getApprovedList();
+
+  return {
+    props: { approvedList },
+  };
+};
+
+const Home = (props: HomeProps) => {
+  console.log(props);
```
Comment (ramonxm) [inline src/pages/index.tsx:45]: remover log

File: src/services/getSiteData.ts [hunk 2]
```diff
@@ -24,3 +24,11 @@ export const getSiteData = async <T>({
 
   return data;
 };
+
+export const getApprovedList = async () => {
+  const response = await fetch(`${config.ADMIN_URL}/platform/approved`).then(
+    (data) => data.json()
```
Comment (ramonxm) [inline src/services/getSiteData.ts:30]: isso aqui funcionou? o certo seria

```js
.then(async (data) => await data.json())
```

**Review by ramonxm — CHANGES_REQUESTED** [review]

**Comment by ramonxm** [comment]
Não sei se isso aqui é um draft, mas o build falhou:
https://github.com/estartando-devs/site/runs/5794497907?check_suite_focus=true

**Review by henriquecalenzoo — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (yagovaluchedevs) [inline src/pages/index.tsx]: remover o ponto e parênteses

**Review by jonataspinto — APPROVED** [review]

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.72 | ✅ |
| First Contentful Paint | 1.3 s |  |
| Time to Interactive | 7.7 s |  |
| Speed Index | 2.2 s |  |
| Total Blocking Time | 690 ms |  |
| Largest Contentful Paint | 3.0 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.85 | ✅ |
| First Contentful Paint | 2.3 s |  |
| Time to Interactive | 6.9 s |  |
| Speed Index | 3.0 s |  |
| Total Blocking Time | 190 ms |  |
| Largest Contentful Paint | 2.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? **GitHub runners could be the cause.**
[Try running on Foo instead](https://www.foo.software/docs/lighthouse-check-github-action/examples#running-on-foo-and-saving-results)

**Comment by estartandodevs** [comment]
**Size Change:** 0 B 🆕

**Total Size:** 0 B



<a href="https://github.com/preactjs/compressed-size-action"><sub>compressed-size-action</sub></a>

**Review by henriquecalenzoo — APPROVED** [review]

**Pull request merged**